src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1225,6 +1237,40 @@ class ContentExplorer extends Component<Props, State> {
         this.setState({ currentOffset: newOffset }, this.refreshCollection);
     };
 
+    /**
+     * Renderer used for cards in grid view
+     *
+     * @param {number} slotIndex - index of item in currentCollection.items
+     * @return {React.Element} - Element to display in card
+     */
+    slotRenderer = (slotIndex: number) => {
+        const { currentCollection, view } = this.state;
+        const { canPreview, canShare, canDownload, canDelete, canRename, isSmall, isTouch, rootFolderId } = this.props;
+        const item: ?BoxItem = getProp(currentCollection, `items[${slotIndex}]`);
+
+        return (
+            <GridViewItem
+                canDelete={canDelete}
+                canDownload={canDownload}
+                canPreview={canPreview}
+                canRename={canRename}
+                canShare={canShare}
+                isSmall={isSmall}
+                isTouch={isTouch}
+                item={item}
+                onItemClick={this.onItemClick}
```

Comment:
It looks like some of these props are duplicated from those passed into `Content`. Can `Content` pass the props down to its children directly?